numpy/add_newdocs.py

```diff
@@ -2150,14 +2152,16 @@ def luf(lamdaexpr, *args, **kwargs):
             like float64 to float32, are allowed.
           * 'unsafe' means any data conversions may be done.
 
+        Default is 'safe'.
+
     Returns
     -------
     output : ndarray
         The calculation based on the Einstein summation convention.
 
     See Also
     --------
-    dot, inner, outer, tensordot
+    einsum, dot, inner, outer, tensordot
 
     Notes
     -----
```

Comment:
IIRC, the examples need changing, but I can do that later. I expect this function might be changing a bit down the line in any case.